.circleci/config.yml

```diff
@@ -27,18 +27,10 @@ commands:
             EXPECTED_SIGNATURE=$(curl -s https://composer.github.io/installer.sig)
             ACTUAL_SIGNATURE=$(php -r "echo hash_file('sha384', 'composer-setup.php');")
             [[ "$EXPECTED_SIGNATURE" == "$ACTUAL_SIGNATURE" ]] && sudo php composer-setup.php --install-dir=/bin --filename=composer || exit 1
-
-  conditionally-skip:
-    description: "Skip if the current commit has no change in relevant directories"
-    parameters:
-      directories:
-        type: string
+  nodegit-workaround:
+    description: "Ensure nodegit runs without error"
     steps:
-      - run:
-          name: Checking whether relevant files were changed
-          command: |
-            CHANGED_FILES="$( git diff HEAD^ HEAD --name-only --diff-filter=ACMRT << parameters.directories >> )"
-            if [ -z "$CHANGED_FILES" ] && [ -z "$CIRCLE_TAG" ]; then echo "No change to relevant file, exiting job" && circleci-agent step halt; fi
+      - run: sudo apt-get update && sudo apt-get install libkrb5-dev
```

Comment:
Ported from https://github.com/studiopress/genesis-custom-blocks/pull/43 and https://github.com/studiopress/genesis-custom-blocks/pull/44, various fixes to prevent `npm ci` errors